### PR TITLE
Add support for PHP 8.2, remove support for PHP 7.4, remove prophecy usage

### PR DESCRIPTION
**Pull request by gsteel**
|    Q          |   A
|-------------- | ------
| New Feature   | yes

### CODE REVIEW

**Review by Ocramius — APPROVED** [review]
Excellent, thanks @gsteel!

**Comment by Ocramius** [comment]
Ah, I see @Slamdunk is handling this 👍

File: composer.json [hunk 1]
```diff
@@ -25,24 +25,23 @@
     "require-dev": {
         "ext-intl": "*",
         "doctrine/annotations": "^1.13.3",
-        "laminas/laminas-captcha": "^2.11.0",
```
Comment (Slamdunk) [inline composer.json:28]: Can you update the `conflict` section as well please?

**Review by Slamdunk — CHANGES_REQUESTED** [review]
Just a minor adjustment to `composer.json` and we can go :muscle:

**Review by Slamdunk — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (gsteel) [reply composer.json:28]: Assuming you want the conflicts to prevent installation of anything less than most recent? Why is this necessary?

Comment (Slamdunk) [reply composer.json:28]: Only tested versions are supported.
Without the `conflict` section:

1. a user may install an unsupported version
2. find a bug
3. report an Issue here
4. we close it with "Bump your deps"

`conflict` prevents this all

Comment (gsteel) [reply composer.json:28]: Thanks for explaining @Slamdunk 👍

**Pull request merged**